README.md

```diff
@@ -26,7 +26,13 @@ import { AuthProvider } from '@descope/react-sdk';
 
 const AppRoot = () => {
 	return (
-		<AuthProvider projectId="my-project-id">
+		<AuthProvider
+			projectId="my-project-id"
```

Comment:
Nice, I would also add a dedicated section for this in the readme, that points to the code sample.

Reply:
Yes, but in this snippet people might miss this, maybe we should have a section so it will be more visible

Reply:
done